### PR TITLE
Test Load Function Online

### PR DESCRIPTION
**Pull request by kartikdutt18**
So across all PRs yesterday LoadSaveTest Failed. I tried it on my Mac (OS Sierra and Catalina) one with Julia bindings and the other with python bindings. None of them failed. So I just want to check if the tests fail. Hopefully they don't and I close this :) .
Thanks.

### CODE REVIEW

**Comment by kartikdutt18** [comment]
Hmm, The test still failed, I think this warrants more attention.

**Comment by bisakhmondal** [comment]
Hi @kartikdutt18 , quick & nice observation👍. I submitted a PR the day before yesterday, but there was no LoadSaveTest failure. Also no new PR merge. So why? I don't get it... 
Thanks.

**Comment by kartikdutt18** [comment]
Hi @bisakhmondal, this might be because of new version of dependency that mlpack uses. Currently working on finding that out. Locally it works fine.

**Comment by kartikdutt18** [comment]
Yup it's related to a dependency, I ran update across all my dependencies and finally got the test to fail.

File: .ci/macos-steps.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ steps:
     pip install cython numpy pandas zipp
     brew update
     brew install openblas armadillo boost
+    brew upgrade hdf5
```
Comment (zoq) [inline .ci/macos-steps.yaml]:
```suggestion
    brew install openblas boost
    brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/9620946e004a29744f9c5e4a27800cba6d6d6b9b/Formula/armadillo.rb
```

Comment (kartikdutt18) [reply .ci/macos-steps.yaml]: Hi @zoq, So I tried this this version and even some earlier version (I think till 9.800.4) and they gave me another build error same as the one on the pipeline i.e. pp:26:14:
```
 fatal error: '/usr/local/Cellar/hdf5/1.10.6/include/hdf5.h' file not found
    #include ARMA_INCFILE_WRAP(ARMA_HDF5_HEADER)
```
The only version that didn't give this error was 9.850.1_1 (current) but that fails load and save test. The test fails for all hd5 formats i.e. h5, hdf5 and so on.

**Comment by kartikdutt18** [comment]
Closing this in lieu of #2278.

**Pull request closed without merging**